docs/examples/account-alias-ecdsa.js

```diff
@@ -0,0 +1,111 @@
+     * to a Hedera account.
+     *
+     * An AccountId takes one of two forms: a normal AccountId with a null aliasKey member takes the form 0.0.123,
+     * while an account ID with a non-null aliasKey member takes the form
+     * 0.0.302a300506032b6570032100114e6abc371b82dab5c15ea149f02d34a012087b163516dd70f44acafabf7777
+     * Note the prefix of "0.0." indicating the shard and realm.  Also note that the aliasKey is stringified
+     * as a hex-encoded ASN1 DER representation of the key.
+     *
+     * An AccountId with an aliasKey can be used just like a normal AccountId for the purposes of queries and
+     * transactions, however most queries and transactions involving such an AccountId won't work until Hbar has
+     * been transferred to the aliasKey account.
+     *
+     * There is no record in the Hedera network of an account associated with a given aliasKey
+     * until an amount of Hbar is transferred to the account.  The moment that Hbar is transferred to that aliasKey
+     * AccountId is the moment that that account actually begins to exist in the Hedera ledger.
+     */
+
+    console.log('"Creating" a new account');
+
+    const privateKey = PrivateKey.generateECDSA();
+    const publicKey = privateKey.publicKey;
+
+    // Assuming that the target shard and realm are known.
+    // For now they are virtually always 0 and 0.
+    const aliasAccountId = publicKey.toAccountId(0, 0);
+
+    /*
+     * Note that no queries or transactions have taken place yet.
+     * This account "creation" process is entirely local.
+     */
+
+    console.log("Transferring some Hbar to the new account");
+    const response = await new TransferTransaction()
+        .addHbarTransfer(client.operatorAccountId, new Hbar(10).negated())
+        .addHbarTransfer(aliasAccountId, new Hbar(10))
+        .execute(client);
+    await response.getReceipt(client);
+
+    const balance = await new AccountBalanceQuery()
+        .setNodeAccountIds([response.nodeId])
+        .setAccountId(aliasAccountId)
+        .execute(client);
+
+    console.log(`Balances of the new account: ${balance.toString()}`);
+
+    const info = await new AccountInfoQuery()
+        .setNodeAccountIds([response.nodeId])
+        .setAccountId(aliasAccountId)
+        .execute(client);
+
+    console.log(`Info about the new account: ${info.toString()}`);
+
+    /*
+     * Note that once an account exists in the ledger, it is assigned a normal AccountId, which can be retrieved
+     * via an AccountInfoQuery.
+     *
+     * Users may continue to refer to the account by its aliasKey AccountId, but they may also
+     * now refer to it by its normal AccountId
+     */
+
+    console.log(`The normal account ID: ${info.accountId.toString()}`);
+    console.log(`The alias key: ${info.aliasKey.toString()}`);
+    console.log(`The alias raw key: ${info.aliasKey.toStringRaw()}`);
+    console.log(`The raw private key: ${privateKey.toStringRaw()}`);
```

Comment:
Let's add a log that is the print out of the last 64chars of the private key that can be used in wallets.
Could call it the import key

Reply:
changed the message to detail that the raw private key is to be used for importing